libraries/src/Form/Field/ListField.php

```diff
@@ -231,6 +231,27 @@ public function addOption($text, $attributes = [])
         return $this;
     }
 
+    /**
+     * Method to remove the trashed option from the list field.
+     *
+     * @return  ListField  For chaining.
+     *
+     * @since   3.7.0
```

Comment:
This should be the current release. But we have some magic here so you dont have to keep changing the value if the merge is delayed etc

```suggestion
     * @since   __DEPLOY_VERSION__
```